sentry-android-core/src/main/java/io/sentry/android/core/FrameMetrics.java

```diff
@@ -0,0 +1,102 @@
+package io.sentry.android.core;
+
+import org.jetbrains.annotations.ApiStatus;
+
+@ApiStatus.Internal
+final class FrameMetrics {
```

Comment:
Can you rename it to `SentryFrameMetrics`, to distinguish it from the Android's one?